### DISCUSSION TITLE
Api Paging

### DISCUSSION THREAD

**Discussion by pusters** [sanderkl/PSMeraki#29] [Q&A]
Is there any built in pagination in this?

**Comment by sanderkl** [comment]
Not at the moment, I can add it in a new version if I can replicate the issue.
Do you have hundreds of networks that requires pagination?

**Comment by pusters** [comment]
yes. Though I found a solution, we have 300+ locations and each one has its
own network. Also, if you use the /organizations/<orgnum>/devices/statuses
to return all devices in the org, it returns only 1000 results when there
are 5k+ devices.
